wms/models/datasets/ugrid.py

```diff
@@ -284,7 +281,29 @@ def getmap(self, layer, request):
                     return self.empty_response(layer, request)
 
                 if request.GET['image_type'] == 'filledcontours':
-                    return mpl_handler.tricontourf_response(tri_subset, data, request)
+                    if np.isnan(np.sum(data)):
```

Comment:
It turns out that differences in performance between the two are quite small. There's about a 7 microsecond difference in favor of `np.isnan(data).any()`. I'll opt for better readability.